demisto_sdk/commands/validate/tests/validators_test.py

```diff
@@ -1157,6 +1159,27 @@ def test_ignore_test_doc_non_pack_file_non_pack(self, file_path: str):
         validate_manager = ValidateManager(check_is_unskipped=False)
         assert validate_manager.ignore_test_doc_non_pack_file(file_path)
 
+    @pytest.mark.parametrize('answer, unsearchable, validator', [(True, True, IncidentFieldValidator),
```

Comment:
```suggestion
    @pytest.mark.parametrize('expected_result, unsearchable, validator', [(True, True, IncidentFieldValidator),
```

I think this name is better for your test